lib/more_core_extensions/core_ext/array/tableize.rb

```diff
@@ -118,7 +119,7 @@ def format_row(row, widths, justifications)
       end
 
       def format_field(field, width, justification)
-        field = field.to_s.gsub(/\n|\r/, '').slice(0, width)
+        field = field.to_s.gsub(/\n|\r|\e\[[^m]+m/, '').slice(0, width)
```

Comment:
please remove this. I would like the escape codes to actually print out

Reply:
okay, so this is the line you said before that was the right one that you wanted me to leave in